2020q4/k8s-bhyve.md

```diff
@@ -0,0 +1,16 @@
+## K8S-bhyve ##
+
+Link:	 [K8S-bhyve](https://k8s-bhyve.convectix.com)
+Link:	 [K8S-bhyve](https://github.com/k8s-bhyve)
+Link:	 [Kubernetes](https://kubernetes.io/)
+
+Contact: Kirill Ponomarev <krion@FreeBSD.org>
+Contact: Oleg Ginzburg <olevole@olevole.ru>
+
+K8S-bhyve is opensource project concentrating primarily to deploy and use
+kubernetes on FreeBSD/bhyve in more agile and more comfortable. We are going
+to provide distributed multi-DC environment or just stand-alone clusters with
+native PV/PVC support.
+
+For 2020q4 we made and published k8s-bhyve image which
+you might install with ISO/memstick, as well as with bsdinstall.
```

Comment:
perhaps provide a link to the memstick, or mention at which of the links above it can be found?